<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QContainerWidget</class>
 <widget class="QWidget" name="QContainerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" alignment="Qt::AlignTop">
       <widget class="QWidget" name="ContWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="1" column="0">
          <widget class="QLabel" name="ContNameLabel">
           <property name="text">
            <string>ContName</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="4">
          <widget class="QGroupBox" name="ContGroupBoxNet">
           <property name="title">
            <string>Connected Networks</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QFrame" name="ConnNetFrame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="3">
                <widget class="QLineEdit" name="ContNetIpEdit">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>ip</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="ContNetNameLabel">
                 <property name="text">
                  <string>NName</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="5">
                <widget class="QPushButton" name="ContNetApplyBtn">
                 <property name="text">
                  <string>Apply</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QLineEdit" name="ContNetSubnetEdit">
                 <property name="text">
                  <string>subnet</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="ContNetIdLabel">
                 <property name="text">
                  <string>NIdaaaaaa</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="ContNetNumLabel">
                 <property name="text">
                  <string>Network_n</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="ContIdLabel">
           <property name="text">
            <string>Id</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="ContDelBtn">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="ContStopBtn">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="4">
          <widget class="Line" name="ContLine">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="ContStartBtn">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="ContTermOpenBtn">
           <property name="text">
            <string>Open in terminal</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="ContStatusLabel">
           <property name="text">
            <string>Status</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="ContGroupBoxRoute">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Routing</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0" colspan="4">
          <widget class="QTextBrowser" name="ContRTextBrowser"/>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="ContBtnRAdd">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="ContBtnRDel">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QPushButton" name="ContBtnRRefresh">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
